lua/spec/helpers/test_helper.lua

```diff
@@ -103,20 +103,57 @@ return function(busted, helper, options)
 		file:close()
 	end
 
+	-- Simplified MediaWiki parser for testing purposes
+	local function simpleMediaWikiParser(text)
+		-- Italics and bold
+		text = text:gsub("'''(.-)'''", '<b>%1</b>')
+		text = text:gsub("''(.-)''", '<i>%1</i>')
+		-- Images
+		for imgParams in text:gmatch('%[%[File:(.-)%]%]') do
+			local params = mw.text.split(imgParams, '|', true)
+			local img = params[1]
+			local width, height, link, alt
+			for i = 2, #params do
+				if params[i]:match('^(%d*)x*(%d*)px$') then
+					width, height = params[i]:match('^(%d*)x*(%d*)px$')
+				elseif params[i]:match('^link=') then
+					link = params[i]:gsub('^link=', '')
+				elseif params[i]:match('^alt=') then
+					alt = params[i]:gsub('^alt=', '')
+				end
+			end
+			width = width or 100
+			height = height or 100
+			alt = alt or ''
+			link = link or img
+			text = text:gsub('%[%[File:' .. imgParams .. '%]%]', '<img src="https://placehold.co/' .. width .. 'x' .. height .. '?text=' .. alt ..'\n'.. link ..'"/>')
+		end
+		-- Internal Urls
+		text = text:gsub('%[%[(.-)|(.-)%]%]', '<a href="%1">%2</a>')
+		text = text:gsub('%[%[(.-)%]%]', '<a href="%1">%1</a>')
```

Comment:
- what about inter wiki links? (e.g.: `[[lpcommons:File_Standards_Guide]]`)
- what about links to files? (e.g. `[[:File:Liquipedia_icon.svg]]`)